tests/tests/test_train_schedule.py

```diff
@@ -741,7 +741,7 @@ def test_etcs_schedule_result_slowdowns(
     _assert_equal_speeds(speed_after_brake_288_to_142, SPEED_LIMIT_142)
 
     # Second slowdown
-    offset_start_brake_142_to_120 = 44_413_857
+    offset_start_brake_142_to_120 = 44_413_825
```

Comment:
Initially wrongly named:
```suggestion
    offset_start_brake_142_to_112 = 44_413_825
```

Reply:
Fixed in https://github.com/OpenRailAssociation/osrd/pull/11616/commits/d328ef56112a6bcbb3a997d5966f1b8f63722afa